components/alert.html

```diff
@@ -0,0 +1,73 @@
+</head>
+<body>
+    <div class="container">
+        <div class="nav">
+            <img src="../img/logo.png" height="80px"></a>
+            <ul>
+                <li><a href="../index.html">Home</a></li>
+                <li><a href="../components/docs.html">Documentation</a></li>
+                <li class="gray-text">v0.1</li>
+                <li><a class="link" href="https://github.com/ashrayparmar/Copmonent-Library"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+            </ul>
+        </div>
+
+        <div class="aside-container">
+            <div class="aside">
+                <ul>
+                    <li><a class="link" href="../components/docs.html">Getting Started</a></li>
+                    <li><a class="link" href="../components/avatar.html">Avatar</a></li>
+                    <li><a class="link-active" href="../components/alert.html">Alert</a></li>
+                    <li><a class="link" href="../components/badge.html">Badge</a></li>
+                    <li><a class="link" href="../components/button.html">Buttons</a></li>
+                    <li><a class="link" href="../components/cards.html">Cards</a></li>
+                    <li><a class="link" href="../components/image.html">Image</a></li>
+                    <li><a class="link" href="../components/input.html">Input</a></li>
+                    <li><a class="link" href="../components/list.html">List</a></li> 
+                    <li><a class="link" href="../components/navigation.html">Navigation Bar</a></li>
+                    <li><a class="link" href="../components/modal.html">Modal</a></li>
+                    <li><a class="link" href="../components/rating.html">Rating</a></li>
+                    <li><a class="link" href="../components/slider.html">Slider</a></li>
+                    <li><a class="link" href="../components/grid.html">Simplified Grid</a></li>
+                    <li><a class="link" href="../components/text.html">Text Utilities</a></li>
+                    <li><a class="link" href="../components/toast.html">Toast</a></li>
+                </ul>
+            </div>
+        </div>
+
+        <div class="content">
+                        
+            <div class="avatar-component">
+            <h2>Alert</h2>
+            <h3>It provide flexible alert messages with respect to user's actions</h3>
+            <hr/>
+
+            <div class="alert-container">
+                <div class="alerts success"><span>File upload succesfull</span></div>
+                <div class="alerts failed"><span>File upload failed</span></div>
+                <div class="alerts warning"><span>Your item's in the cart will expire in 5 minutes</span></div>
+                <div class="alerts information"><span>Last date to pay the bill is tomorrow</span></div>
+            </div>
+            
+            <h4 style="color: gray; padding: 1rem 1rem; font-weight: 300;">To use Alert component copy the below HTML link</h4> 
+            <iframe
```

Comment:
while import iframe from carbon.sh.now you can change the background color to white from gray.  it will look good.